splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/UpgradeConglomerateTable.java

```diff
@@ -0,0 +1,42 @@
+package com.splicemachine.derby.impl.sql.catalog.upgrade;
+
+import com.splicemachine.access.api.PartitionAdmin;
+import com.splicemachine.access.configuration.HBaseConfiguration;
+import com.splicemachine.db.iapi.error.StandardException;
+import com.splicemachine.db.iapi.store.access.TransactionController;
+import com.splicemachine.derby.impl.sql.catalog.SpliceDataDictionary;
+import com.splicemachine.si.impl.driver.SIDriver;
+import com.splicemachine.utils.SpliceLogUtils;
+
+import java.io.IOException;
+
+public class UpgradeConglomerateTable extends UpgradeScriptBase {
+    public UpgradeConglomerateTable(SpliceDataDictionary sdd, TransactionController tc) {
+        super(sdd, tc);
+    }
+
+    @Override
+    protected void upgradeSystemTables() throws StandardException {
+        SpliceLogUtils.info(LOG, "Creating SPLICE_CONGLOMERATE_SI table...");
+        PartitionAdmin admin = null;
+        try {
+            admin = SIDriver.driver().getTableFactory().getAdmin();
+            if (!admin.tableExists(HBaseConfiguration.CONGLOMERATE_SI_TABLE_NAME)) {
+                admin.createSITable(HBaseConfiguration.CONGLOMERATE_SI_TABLE_NAME);
+                UpgradeUtils.initializeConglomerateSITable(tc);
+            }
+        }
+        catch (IOException e) {
```

Comment:
Catch any kind of exception here just in case

Reply:
Done